src/components/synchronization-page/GenericSynchronizationPage.jsx

```diff
@@ -39,6 +39,8 @@ class GenericSynchronizationPage extends React.Component {
     };
 
     changeSelection = metadataIds => {
+        const { clearSelection } = this.props;
+        if (metadataIds.length === 0) clearSelection();
```

Comment:
It should be added to `propTypes`. However, I an not sure about this prop, couldn't we use a more generic `onSelectionChange`? This is related with the comment in the PR: 

> Due to this limitation, you might lose part of the selection when using this action (selection in other pages)

MetadataTable has `notifyNewSelection`, can't we use it also in `GenericSynchronizationPage` so `OrganisationUnitPage` knows the previous selection on `selectChildren`?

Reply:
Yeah, this part is really "tricky".

If we add notifyNewSelection in "OrganisationUnitPage" we will override the one in "GenericSynchronizationPage". My inner java would love to have a subscription-like method to attach to selection changes without taking care of how many observers it has.

I would also love to have a method that programmatically selects a collection of ids from outside of the ObjectsTable but we were trying to have a quick fix for this to ship as soon as possible.

I would propose for the next "big" app project to re-work ObjectsTable and allow uncontrolled behavior for the objects and selection collections.

Meanwhile, to make it more generic I will add another onSelectionChange that does not override the other one and add it to the prop-types.